lib/browser/api/touch-bar.js

```diff
@@ -118,6 +118,9 @@ class TouchBar extends EventEmitter {
       window.removeListener('closed', removeListeners)
       window._touchBar = null
       delete this.windowListeners[id]
+      for (const item of this.ordereredItems) {
+        item.removeListener('change', this.changeListener)
```

Comment:
I _think_ this is undoing the `item.on('change', this.changeListener)` in `registerItem()`, but the next line there is to also register each of item.child.ordereredItems iff item.child is a TouchBar. Do we need to undo that here too?

If we don't also need that, then I'm :+1: on this PR

Reply:
_edit 2:_ do we also need to remove the escapeItem changeListener too?

Reply:
I know clean up recursively and deal with the `escapeItem` listener